packages/client/wallets/aa/src/blockchain/wallets/smartAccount.ts

```diff
@@ -0,0 +1,49 @@
+import { logInfo } from "@/services/logging";
+import { gelatoBundlerProperties, usesGelatoBundler } from "@/utils";
+import { SmartAccountClient } from "permissionless";
+import { EntryPoint } from "permissionless/types/entrypoint";
+
+import { EVMBlockchainIncludingTestnet } from "@crossmint/common-sdk-base";
+
+export function toCrossmintSmartAccountClient<Client extends SmartAccountClient<EntryPoint>>({
```

Comment:
Anything missing from the methods we'll delete (the performance logging for example) will be added here in subsequent PRs.